.github/workflows/flowzone.yml

```diff
@@ -0,0 +1,16 @@
+name: Flowzone
+
+on:
+  pull_request:
+    types: [opened, synchronize, closed]
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  flowzone:
+    name: Flowzone
+    uses: product-os/flowzone/.github/workflows/flowzone.yml@master
+    secrets: inherit
+    with:
+      balena_slugs: balenalabs/balena-sense
```

Comment:
Looks like this fleet does not exist, or the token we are using doesn't have access?

Reply:
Maybe it doesn't but it's what balenaCI would have pushed to for the `type: balena` repo